Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download.yml

```diff
@@ -0,0 +1,606 @@
+id: Endace Search Archive and Download
+version: -1
+name: Endace Search Archive and Download
+description: This playbook uses Endace APIs to search, archive and download PCAP file
+  from either a single EndaceProbe or many via the InvestigationManager and enables
+  integration of full historical packet capture into security automation workflows.
+starttaskid: '0'
+tasks:
+  '0':
+    id: '0'
+    taskid: 16458520-3127-435e-8eb4-894b42cee07d
+    type: start
+    task:
+      id: 16458520-3127-435e-8eb4-894b42cee07d
+      version: -1
+      name: ''
+      iscommand: false
+      brand: ''
+      description: ''
+    nexttasks:
+      '#none#':
+      - '28'
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 50,\n    \"y\": 50\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '7':
+    id: '7'
+    taskid: a286a4dd-e0f0-48d3-84cf-da04fea39679
+    type: title
+    task:
+      id: a286a4dd-e0f0-48d3-84cf-da04fea39679
+      version: -1
+      name: Done
+      tags:
+      - Success
+      type: title
+      iscommand: false
+      brand: ''
+      description: ''
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 70,\n    \"y\": 2470\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '9':
+    id: '9'
+    taskid: 6fd5a63a-fb21-4236-8fd3-609be563ab95
+    type: condition
+    task:
+      id: 6fd5a63a-fb21-4236-8fd3-609be563ab95
+      version: -1
+      name: Is search progress = 100%?
+      description: Check whether the values provided in arguments are equal. If either
+        of the arguments are missing, no is returned.
+      scriptName: AreValuesEqual
+      type: condition
+      iscommand: false
+      brand: ''
+    nexttasks:
+      no:
+      - '7'
+      yes:
+      - '26'
+    scriptarguments:
+      left:
+        simple: ${Endace.Search.Response.JobProgress}
+      right:
+        simple: '100'
+    results:
+    - AreValuesEqual
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 50,\n    \"y\": 720\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '14':
+    id: '14'
+    taskid: dcdaa108-a3eb-4472-8dc2-e0572f59de60
+    type: condition
+    task:
+      id: dcdaa108-a3eb-4472-8dc2-e0572f59de60
+      version: -1
+      name: Is PCAP FileSize < 50MB?
+      description: "Checks if one number(float) as bigger than the other(float)\n\
+        Returns yes: if first > second\nReturns no: if first <= second\nReturns exception\
+        \ if one of the inputs is not a number"
+      scriptName: IsGreaterThan
+      type: condition
+      iscommand: false
+      brand: ''
+    nexttasks:
+      no:
+      - '7'
+      yes:
+      - '35'
+    scriptarguments:
+      first:
+        simple: '50000'
```

Comment:
Is this suppose to be in bytes?

Reply:
yes

Reply:
So should be 50000000.

Reply:
it should be 50 - for 50 MB. i had 50GB (50000 MB) before. not a bug, but certainly defeats the purpose of this check. I am change it as i create a new playbook.

Reply:
Last time I checked 
![image](https://user-images.githubusercontent.com/51712181/81008846-45f45f80-8e5c-11ea-97de-231c3b82fc52.png)

Reply:
> 
> 
> Is this suppose to be in bytes?
The correct answer is No. in MB.

Reply:
50 MB might be a bit heavy to analyze,10 MB should be nice to start with.
We can always add an option for the user to approve larger files later on

Reply:
Sure. it's user configurable though.